Docker/base-images/entando-postgresql95-base/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM centos/postgresql-95-centos7:9.5
+LABEL io.openshift.s2i.destination="/opt/s2i/destination" \
+      maintainer="Ampie Barnard <a.barnard@entando.com>" \
+      io.k8s.description="Postgresql Database Image for building and running Entando applications" \
+      io.k8s.display-name="Postgresql Database Image for Entando 5+"
+ARG ENTANDO_VERSION_TO_BUILD
+ENV MAVEN_VERSION="3.5.4" \
+    JAVA_HOME="/usr/lib/jvm/java-1.8.0" \
```

Comment:
Are maven and java needed to build the postgres db?

Reply:
Yes indeed. We run mvn jetty:run in the background to build the database.